app/views/spotlight/catalog/_document_admin_table.html.erb

```diff
@@ -11,7 +11,7 @@
   </thead>
 
   <% if Blacklight.version < '8.0' %>
-    <%= render view_config.document_component.with_collection(documents) %>
+    <%= render (view_config.document_component || Spotlight::DocumentAdminTableComponent).with_collection(documents) %>
```

Comment:
I'm curious as to why a fallback is needed here since you set the component in CatalogController.

Reply:
Besides what I put in the PR description?

We had a pre-existing customization in exhibits that broke the admin page without some other change (e.g. https://github.com/sul-dlss/exhibits/pull/2574/commits/7666bf83b52c1c283515226de43920dcb9d5b9d6), and that's something we should try to avoid doing to downstream applications?

Reply:
Yeah, I'm still not understanding how `view_config.document_component` could be nil. since it was set in CatalogController in this PR.   Wouldn't that change alone have prevented the problem we had in Exhibits?

Reply:
If the partial is rendered outside of the `catalog#admin` action, the contents of the `before_action` block aren't evaluated.

Reply:
Okay. My assumption was that no one would ever do that.